nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -452,63 +457,66 @@
 
-- The `services.mxisd` module has been removed as both [mxisd](https://github.com/kamax-matrix/mxisd) and [ma1sd](https://github.com/ma1uta/ma1sd) are not maintained any longer.
-  Consequently the package `pkgs.ma1sd` has also been removed.
+- `services.mxisd` has been removed as both [mxisd](https://github.com/kamax-matrix/mxisd) and [ma1sd](https://github.com/ma1uta/ma1sd) are no longer maintained.
+  Consequently, the package `ma1sd` has also been removed.
+
+- `mikutter` has been removed, as the package was broken and had no maintainers in nixpkgs.
 
 - `ffmpeg_5` has been removed. Please use the unversioned `ffmpeg`,
-  pin a newer version, or if necessary pin `ffmpeg_4` for compatibility.
+  pin a newer version, or pin `ffmpeg_4` for compatibility if necessary.
 
-- The `xdg.portal.gtkUsePortal` option has been removed, as it had been deprecated for over 2 years. Using the `GTK_USE_PORTAL` environment variable in this manner is not intended nor encouraged by the GTK developers, but can still be done manually via `environment.sessionVariables`.
+- `xdg.portal.gtkUsePortal` has been removed, as it had been deprecated for over 2 years. Using the `GTK_USE_PORTAL` environment variable in this manner is not intended nor encouraged by the GTK developers, but can still be done manually via `environment.sessionVariables`.
 
-- The `services.trust-dns` module has been renamed to `services.hickory-dns`.
+- `services.trust-dns` has been renamed to `services.hickory-dns`.
 
-- The option `services.prometheus.exporters.pgbouncer.connectionStringFile` has been removed since
+- `services.prometheus.exporters.pgbouncer.connectionStringFile` has been removed since
   it leaked the connection string (and thus potentially the DB password) into the cmdline
   of process making it effectively world-readable.
 
   Use [`services.prometheus.exporters.pgbouncer.connectionEnvFile`](#opt-services.prometheus.exporters.pgbouncer.connectionEnvFile) instead.
 
-- The `lsh` package and the `services.lshd` module have been removed as they had no maintainer in Nixpkgs and hadn’t seen an upstream release in over a decade. It is recommended to migrate to `openssh` and `services.openssh`.
+- `lsh` and `services.lshd` have been removed as they had no maintainer in Nixpkgs and no upstream release in over a decade. It is recommended to migrate to `openssh` and `services.openssh`.
 
 - `opencv2` and `opencv3` have been removed, as they are obsolete and
   were not used by any other package. External users are encouraged to
   migrate to OpenCV 4.
 
-- The `tvheadend` package and the `services.tvheadend` module have been
-  removed as nobody was willing to maintain them and they were stuck on
-  an unmaintained version that required FFmpeg 4; please see [pull
+- `tvheadend` package and the `services.tvheadend` module have been
+  removed due to lack of maintenance in nixpkgs and being stuck on
+  a version that required FFmpeg 4. Please see the related [pull
```

Comment:
The version that requires FFmpeg 4 is from a release branch that is unmaintained upstream, so I think removing “unmaintained” here loses some nuance.